components/multiselect/multiselect.tsx

```diff
@@ -240,6 +247,33 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
         this.props.handleDelete(values);
     }
 
+    MultiValueRemove = ({children, innerProps}: any) => (
+        <div {...innerProps}>
+            {children || <CloseCircleSolidIcon className='h-4 w-4 d-flex mr-1'/>}
+        </div>
+    );
+
+    formatOptionLabel = (user: any) => {
+        const profileImg = imageURLForUser(user.id, user.last_picture_update);
+
+        return (
+            <React.Fragment>
+                <Avatar
+                    size='sm'
+                    username={user.username}
+                    url={profileImg}
+                />
+                <div style={{margin: '0 2px 0 8px'}}>
```

Comment:
Let's not use inline styles as we are getting a lot of conflicts because of the existing ones.